Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -208,85 +211,87 @@ void doEsirkepovDepositionShapeN (const amrex::ParticleReal * const xp,
                                   const amrex::Real q,
                                   const long n_rz_azimuthal_modes)
 {
+    using namespace amrex;
+
     // Whether ion_lev is a null pointer (do_ionization=0) or a real pointer
     // (do_ionization=1)
-    const bool do_ionization = ion_lev;
-    const amrex::Real dxi = 1.0/dx[0];
-    const amrex::Real dtsdx0 = dt*dxi;
-    const amrex::Real xmin = xyzmin[0];
+    bool const do_ionization = ion_lev;
+    Real const dxi = 1.0_rt / dx[0];
+    Real const dtsdx0 = dt*dxi;
+    Real const xmin = xyzmin[0];
 #if (defined WARPX_DIM_3D)
-    const amrex::Real dyi = 1.0/dx[1];
-    const amrex::Real dtsdy0 = dt*dyi;
-    const amrex::Real ymin = xyzmin[1];
+    Real const dyi = 1.0_rt / dx[1];
+    Real const dtsdy0 = dt*dyi;
+    Real const ymin = xyzmin[1];
 #endif
-    const amrex::Real dzi = 1.0/dx[2];
-    const amrex::Real dtsdz0 = dt*dzi;
-    const amrex::Real zmin = xyzmin[2];
+    Real const dzi = 1.0_rt / dx[2];
+    Real const dtsdz0 = dt*dzi;
+    Real const zmin = xyzmin[2];
 
 #if (defined WARPX_DIM_3D)
-    const amrex::Real invdtdx = 1.0/(dt*dx[1]*dx[2]);
-    const amrex::Real invdtdy = 1.0/(dt*dx[0]*dx[2]);
-    const amrex::Real invdtdz = 1.0/(dt*dx[0]*dx[1]);
+    Real const invdtdx = 1.0_rt / (dt*dx[1]*dx[2]);
+    Real const invdtdy = 1.0_rt / (dt*dx[0]*dx[2]);
+    Real const invdtdz = 1.0_rt / (dt*dx[0]*dx[1]);
 #elif (defined WARPX_DIM_XZ) || (defined WARPX_DIM_RZ)
-    const amrex::Real invdtdx = 1.0/(dt*dx[2]);
-    const amrex::Real invdtdz = 1.0/(dt*dx[0]);
-    const amrex::Real invvol = 1.0/(dx[0]*dx[2]);
+    Real const invdtdx = 1.0_rt / (dt*dx[2]);
+    Real const invdtdz = 1.0_rt / (dt*dx[0]);
+    Real const invvol = 1.0_rt / (dx[0]*dx[2]);
 #endif
 
 #if (defined WARPX_DIM_RZ)
-    const Complex I = Complex{0., 1.};
+    Complex const I = Complex{0., 1.};
 #endif
 
-    const amrex::Real clightsq = 1.0/PhysConst::c/PhysConst::c;
+    Real const clightsq = 1.0_rt / ( PhysConst::c * PhysConst::c );
```

Comment:
Only algorithmic change: this causes the point-wise regression test on the MC process check of ionization to indicate a false-positive error.